packages/server/tests/acceptance.spec.ts

```diff
@@ -559,10 +625,28 @@ describe('RPC Server Integration Tests', async function () {
     };
 
     const numberTo0x = (input: number): string => {
-        return `0x${input.toString(16)}`;
+        return `0x${toHex(input)}`;
     };
 
     const prune0x = (input: string): string => {
         return input.startsWith('0x') ? input.substring(2) : input;
     };
+
+    const toHex = (num) => {
```

Comment:
nit: All the utils methods can be added in a Utils class to be usable outside the acceptance.spec.ts